ArduPlane/altitude.cpp

```diff
@@ -30,7 +30,7 @@ void Plane::adjust_altitude_target()
 
 void Plane::check_home_alt_change(void)
 {
-    int32_t home_alt_cm = ahrs.get_home().alt;
+    int32_t home_alt_cm = ahrs.get_home().get_alt_cm();
```

Comment:
this could be done using get_alt_m() and it needs to be passed to TECS as _m anyway